uint8 x
uint8 y
float32 yaw
---
nav_msgs/Path path